src/AssemblerOptions.cpp

```diff
@@ -201,6 +203,11 @@ void AssemblerOptions::addConfigurableOptions()
         default_value(10000),
         "Read length cutoff. Shorter reads are discarded.")
 
+        ("Reads.desiredCoverage",
+        value<string>(&readsOptions.desiredCoverageString)->
+        default_value("0"),
+        "Desired coverage as a number of bases. Coverage will be reduced to this if necessary.")
```

Comment:
I will update the documentation and `shasta.conf` accordingly, and possibly expand the message a bit.